addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2202,6 +2223,18 @@ export class Wysiwyg extends Component {
             button.classList.toggle('active', button.dataset.value === value);
         }
     }
+    updateTextWrappingButtons(buttons, image) {
+        if (image) {
+            const inline = !(image.classList.contains('float-start') || image.classList.contains('float-end') ||
+                            image.classList.contains('mx-auto') || image.classList.contains('d-block'));
+            const wrapText = image.classList.contains('float-start') || image.classList.contains('float-end');
+            const breakText = image.classList.contains('mx-auto') || image.classList.contains('d-block');
+
+            buttons[0] && buttons[0].classList.toggle('active', inline);
+            buttons[1] && buttons[1].classList.toggle('active', wrapText);
+            buttons[2] && buttons[2].classList.toggle('active', breakText);
```

Comment:
Is it possible that `buttons[x]` is falsy? :thinking: In any case, I would never do this: using short-circuiting should be reserved for assignments and conditions IMO: especially since in general, if you have the possibility of having a falsy object on which you want to perform some action, you need to explain why with a comment.

Reply:
Yeah, I think we skipped removing this part of code while updation. It was breaking because in my previous code it gives me some falsy values there. :sweat_smile: